spec/factories/furniture/marketplace.rb

```diff
@@ -65,6 +65,14 @@
       with_delivery_areas
       with_notification_methods
     end
+
+    trait :with_square do
```

Comment:
Is there supposed to be an Application ID? I'm adding secrets to Github and I've added:
 
1. `MARKETPLACE_VENDOR_SQUARE_ACCESS_TOKEN`
2. `MARKETPLACE_VENDOR_SQUARE_LOCATION`
3. `SQUARE_ENV`
4. `MARKETPLACE_DISTRIBUTOR_APPLICATION_ID`

Reply:
I think the application id is only important if we make it an OAUTH App; which I would love for us to do at some point (so that someone can just add "Delivery by Piikup" to their Square account and magic-pony complete!)

Reply:
It's not necessary for instantiating a Square Client class.